core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -3251,6 +3251,44 @@ paths:
           required: true
           schema:
             $ref: '#/definitions/DataRepoSnapshot'
+    get:
+      requestBody:
+        description: |
+          Parameters for enumerating snapshots.
+        content:
+          application/json:
+            schema:
+              $ref: '#/definitions/EnumerateSnapshotRequestBody'
```

Comment:
1. I think `requestBody` is OpenAPI 3, not supported in OpenAPI 2. This should move to `parameters` with `in: body`
2. Is this really a GET request with a body? We should avoid those, and move offset/limit to query params, assuming this remains as a GET

Reply:
I think you're right - moved these to parameters.
I can also just use query params, so I switched to those.